### DISCUSSION TITLE
homebrew cask for desktop app

### DISCUSSION THREAD

**Discussion by onexbash** [outline/outline#7050] [Show and tell]
Hi everyone, 
I just opened a PR with a **homebrew cask** for the outline **desktop app**. 
As soon as it's accepted, you can install outline using brew:
`brew install --cask getoutline`

**Pull Request**
https://github.com/Homebrew/homebrew-cask/pull/176888
**Homebrew Repository**
https://github.com/Homebrew/homebrew-cask

**Comment by tommoor** [comment]
Nice, thanks for the help!
